### PR TITLE
refactor layouts to be part of graph

### PR DESCRIPTION
**Pull request by mariakleiner**
and delete old nodegraph/ app

### CODE REVIEW

File: pkg/Library/Designer/Designer.js [hunk 1]
```diff
@@ -184,15 +182,18 @@ onNodeDelete({eventlet: {key}, graph}, state) {
   };
 },
 
-onNodePosition({eventlet: {key, value}, graph, layout}, state) {
+onNodePosition({eventlet: {key, value}, graph, layoutId}, state) {
   const node = this.findNodeByParticle(key, graph, state.recipes);
   if (!node) {
     return this.selectNode(null, state);
   }
+  graph.layout ??= {};
```
Comment (sjmiles) [inline pkg/Library/Designer/Designer.js:190]: I believe these three lines are equivalent to:

`((graph.layout ??= {})[layoutId] ?? = {})[node.id] = value;`

File: pkg/Library/NodeTree/NodeTree.js [hunk 2]
```diff
@@ -155,15 +155,18 @@ async onNodeSelect({eventlet: {key}}) {
...
 },
 
-async onDrop({eventlet: {key: container, value: {id}}, graph, nodeTypes, layout}, state, {service}) {
+async onDrop({eventlet: {key: container, value: {id}}, graph, nodeTypes, layoutId}, state, {service}) {
   //log('onDrop:', key, container);
   const node = graph.nodes[id];
   const nodeType = nodeTypes[node.type];
   const hostIds = this.getParticleNames(nodeType).map(particleName => this.hostId(node, particleName));
   await service({kind: 'ComposerService', msg: 'setContainer', data: {hostIds, container}});
+  graph.layout ??= {};
```
Comment (sjmiles) [inline pkg/Library/NodeTree/NodeTree.js:164]: As above.

**Review by sjmiles — COMMENTED** [review]
I love this very much. :heart: :heart: :heart:

P.S. I made a couple notes about syntax for "set this deep object value" which has been an annoyer in JS for years. ECMAScript was working on it, we should have a look-see.

File: pkg/Library/Designer/Designer.js (hunk 1 above)
Comment (mariakleiner) [reply pkg/Library/Designer/Designer.js]: great, thanks! done

File: pkg/Library/NodeTree/NodeTree.js (hunk 2 above)
Comment (mariakleiner) [reply pkg/Library/NodeTree/NodeTree.js]: done, thanks!

**Comment by mariakleiner** [comment]
> I love this very much. ❤️ ❤️ ❤️
> 
> P.S. I made a couple notes about syntax for "set this deep object value" which has been an annoyer in JS for years. ECMA was working on it, we should have a look-see.

Thank you! I agree, it is better now. 
Addressed comments, submitting.

**Pull request merged**